docs/DeveloperGuide.md

```diff
@@ -325,6 +325,46 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use Case: Edit an internship entry.**
+
+**MSS**
+
+1.  User edits an internship entry.
+2.  InternBuddy updates that particular internship entry.
+
+    Use case ends
+
+**Extensions**
+
+* 1a. InternBuddy detects that index of internship entry given is not a number.
```

Comment:
Same as line 344.

Reply:
Ok after discussion, will merge 1a and 1b into 1c and 1d since they have much overlap.